src/sentry/static/sentry/app/components/search/index.jsx

```diff
@@ -68,6 +70,8 @@ class Search extends React.Component {
     navigateTo(nextPath, router);
   };
 
+  saveQueryMetrics = debounce(query => analytics('omnisearch.query', {query}), 200);
```

Comment:
hey billy, you'll need to whitelist new events at https://github.com/getsentry/reload/blob/master/reload_app/app.py#L20 before being able to use them. reason is to prevent the server accepting and storing garbage in case of accidental bugs (generally we sanitize/validate any client-side data)

Reply:
would you mind also adding a comment about this to where `analytics()` is defined?